library/core/src/prelude/v1.rs

```diff
@@ -61,11 +61,17 @@ pub use crate::hash::macros::Hash;
 #[allow(deprecated)]
 #[doc(no_inline)]
 pub use crate::{
-    assert, cfg, column, compile_error, concat, concat_idents, env, file, format_args,
-    format_args_nl, include, include_bytes, include_str, line, log_syntax, module_path, option_env,
-    stringify, trace_macros,
+    assert, assert_eq, assert_ne, cfg, column, compile_error, concat, debug_assert, debug_assert_eq, debug_assert_ne, env, file, format_args, include, include_bytes, include_str, line, matches, module_path, option_env, panic, stringify, todo, r#try, unimplemented, unreachable, write, writeln,
 };
 
+#[unstable(feature = "ub_checks", issue = "none")]
+#[doc(no_inline)]
+pub use crate::assert_unsafe_precondition;
```

Comment:
This shouldn't be in the prelude since it is not meant for consumption outside the standard library. Instead, code in the standard library should import this explicitly.